bittensor/_neuron/text/advanced_server/nucleus_impl.py

```diff
@@ -217,6 +217,9 @@ def config ():
         parser.add_argument('--neuron.no_restart', action='store_true', help='if the model should restart', default=False)
         parser.add_argument('--neuron.blacklist.stake.forward', type=float, help='Amount of stake (tao) in order not to get blacklisted for forward requests', default=10)
         parser.add_argument('--neuron.blacklist.stake.backward', type=float, help='Amount of stake (tao) in order not to get blacklisted for backward requests', default=100)
+        parser.add_argument('--neuron.metagraph_sync', type=float, help='how often to sync the metagraph', default=100000)
+        parser.add_argument('--neuron.set_weights', type=float, help='how often to sync set weights', default=10)
```

Comment:
What does this flag mean I dont think it is self explanatory. Also setting weights every 10 blocks is way to fast.

Reply:
Sounds good; let me change it to 100 blocks and update the flags